packages/components/src/tools-panel/styles.ts

```diff
@@ -107,6 +108,15 @@ export const ToolsPanelItem = css`
 			margin-bottom: 0;
 		}
 	}
+
+	/* Standardize InputControl labels with others inside ToolsPanel. */
+	&& ${ LabelWrapper } {
```

Comment:
I ended up using `LabelWrapper` from the input control styles here as I couldn't get `BaseLabel` to work for me. Open to suggestions to improve this.